### PR TITLE
Allow any group that has UserProfile perm to use Email CSV

### PR DESCRIPTION
**Pull request by smarzban**

### CODE REVIEW

**Comment by Mitso** [comment]
@saeedmb8
Could you please add a screenshot of the Admin page.

**Comment by smarzban** [comment]
screen shot 2017-10-30 at 16 53 19

File: tuneme/templates/new/admin/frontend_users_admin_view.html [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+                <div class="left">
+                    <div class="col">
+                        {% block h1 %}<h1 {% if view.header_icon %}class="icon icon-{{ view.header_icon }}"{% endif %}>{{ view.get_page_title }}<span></span></h1>{% endblock %}
+                    </div>
+                    {% block search %}{% search_form %}{% endblock %}
+                </div>
+                <div class="right">
+                  <form method="post" action="">
+                      {% csrf_token %}
+                        <button type="submit" class="button bicolor icon icon-download">{% trans "Email CSV" %}</button>
```
Comment (nathanbegbie) [inline tuneme/templates/new/admin/frontend_users_admin_view.html:28]: Sorry, I'm a bit confused - should this be wrapped in some logic that checks if a user has those permissions, or will they only ever see this template if they have the prerequisite permissions?

**Comment by smarzban** [comment]
@mitso @Nathan so I didn't change the UI, but molo.profile template had the condition to only show the email CSV to superusers https://github.com/praekelt/molo.profiles/blob/develop/molo/profiles/templates/admin/frontend_users_admin_view.html#L28 
I have removed this condition for TuneMe in the template so they can add this in user groups and not limited to the superusers or any specific group

**Comment by nathanbegbie** [comment]
@saeed - so you're saying that the restrictions that you want to place on who can send an email with data is going to be restricted elsewhere? Or has it already been done?

**Comment by nathanbegbie** [comment]
:+1:

**Comment by smarzban** [comment]
@nathanbegbie thanks, yes the restriction will come from the group permissions in CMS

**Pull request merged**